components/infobox/wikis/valorant/infobox_league_custom.lua

```diff
@@ -0,0 +1,246 @@
+		'[[Patch ' .. args.epatch .. '|'.. args.epatch .. ']]'
+	end
+
+	return content
+end
+
+function CustomLeague:_standardiseRawDate(dateString)
+	-- Length 7 = YYYY-MM
+	-- Length 10 = YYYY-MM-??
+	if String.isEmpty(dateString) or (#dateString ~= 7 and #dateString ~= 10) then
+		return ''
+	end
+
+	if #dateString == 7 then
+		dateString = dateString .. '-??'
+	end
+	dateString = dateString:gsub('%-XX', '-??')
+	return dateString
+end
+
+function CustomLeague:_createPrizepool()
+	if String.isEmpty(_args.prizepool) and String.isEmpty(_args.prizepoolusd) then
+		return nil
+	end
+	local date
+	if String.isNotEmpty(_args.currency_rate) then
+		date = _args.currency_date
+	end
+
+	return PrizePoolCurrency._get({
+		prizepool = _args.prizepool,
+		prizepoolusd = _args.prizepoolusd,
+		currency = _args.localcurrency,
+		rate = _args.currency_rate,
+		date = date or Variables.varDefault('tournament_enddate', _TODAY),
+	})
+end
+
+function CustomLeague:_createLiquipediaTierDisplay()
+	local tier = _args.liquipediatier or ''
+	local tierType = _args.liquipediatiertype or ''
+	if String.isEmpty(tier) then
+		return nil
+	end
+
+	local function buildTierString(tierString)
+		local tierText = Tier.text[tierString]
+		if not tierText then
+			table.insert(_league.warnings, tierString .. ' is not a known Liquipedia Tier/Tiertype')
+			return ''
+		else
+			return '[[' .. tierText .. ' Tournaments|' .. tierText .. ']]'
+		end
+	end
+
+	local tierDisplay = buildTierString(tier)
+
+	if String.isNotEmpty(tierType) then
+		tierDisplay = buildTierString(tierType) .. '&nbsp;(' .. tierDisplay .. ')'
+	end
+
+	return tierDisplay
+end
+
+function CustomLeague:defineCustomPageVariables()
+	--Legacy vars
+	Variables.varDefine('tournament_tier', _args.liquipediatier or '')
+	Variables.varDefine('tournament_prizepool', _args.prizepool or '')
+	Variables.varDefine('tournament_name', _args.name or '')
+	Variables.varDefine('tournament_short_name', _args.shortname or '')
+	Variables.varDefine('tournament_ticker_name', _args.tickername or '')
+	Variables.varDefine('special_ticker_name', _args.tickername_special or '')
+	Variables.varDefine('tournament_icon', _args.icon or '')
+	Variables.varDefine('tournament_type', _args.type or '')
+	Variables.varDefine('tournament_series', _args.series or '')
+	Variables.varDefine('tournament_riot_premier', _args.riotpremier or '')
+	Variables.varDefine('tournament_riot_tier', _args.riotpremier or '')
+	Variables.varDefine('tournament_mode', _args.individual or '')
+	Variables.varDefine('patch', _args.patch or '')
+
+	--Legacy date vars
+	local sdate = Variables.varDefault('tournament_startdate', '')
+	local edate = Variables.varDefault('tournament_enddate', '')
+	Variables.varDefine('tournament_sdate', sdate)
+	Variables.varDefine('tournament_edate', edate)
+	Variables.varDefine('tournament_date', edate)
+	Variables.varDefine('date', edate)
+	Variables.varDefine('sdate', sdate)
+	Variables.varDefine('edate', edate)
+end
```

Comment:
vars that were set by the old infobox template but not this module:
- `seriesabbr1`
- `seriesabbr2`
- `metadesc-riot`
- `tournament_region`
- `prizepoollocal`
- `raw_sdate`
- `raw_edate`
- `no team result`